service/worker/scanner/executions/common/interfaces.go

```diff
@@ -27,7 +27,7 @@ import "github.com/uber/cadence/common/persistence"
 type (
 	// PersistenceRetryer is used to retry requests to persistence
 	PersistenceRetryer interface {
-		ListConcreteExecutions(*persistence.ListConcreteExecutionsRequest) (*persistence.ListConcreteExecutionsResponse, error)
+		ListAllExecutions(*persistence.ListConcreteExecutionsRequest) (*persistence.ListConcreteExecutionsResponse, error)
```

Comment:
Maybe just add new API for listAll, otherwise the request and response is a little inconsistent with API name.